### PR TITLE
Updates to documented process

### LINKED ISSUE #49: Add mention of 14/15 when making recommendation
Guides should include mention of hiring 15s, 14s, that we should keep the grade we’re hiring in mind when making hiring recs / selections.

### PR DESCRIPTION
**Pull request by ctro**
Closes #49

### CODE REVIEW

**Comment by ctro** [comment]
I think I forgot this page existed. I made some changes to reflect our current process.
Mostly: Talent team does Resume Reviews now.

File: _pages/detailed-process.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ This helps ensure consistency and fairness into the process. It also lets us col
 Before we start any selection we should:
 
 - Select interviewers (SMEs)
-    - We need 3 SMEs for resume review, and 4 for interviews. These can't be the same people for each candidate. So this points to the need for a fairly large pool. 8-10 is probably a *minimum* for most positions.
+    - We need 4 SMEs for interviews. These can't be the same people for each candidate. So this points to the need for a fairly large pool. 8-10 is probably a *minimum* for most positions.
```
Comment (tbaxter-18f) [inline _pages/detailed-process.md]: This is really minor, but if we're following AP style -- and I think that's handbook preference -- ten and below would be spelled out.

File: _pages/detailed-process.md [hunk 2]
```diff
@@ -119,14 +110,15 @@ Notes on interview format:
...
-    - overlap with previous interviews is just fine (even, good - gives a candidate a chance to recover )
+    - overlap with previous interviews is just fine (even, good - gives a candidate a chance to recover)
 
 ### 6. Decision time
 
-- Interviewers submit feedback in writing to TL+HM (see guide below for format)
-- If the results are especially controversial, hold a meeting to discuss
-- HM makes final hire/no-hire decision
-    - Consensus isn't required, nor is there any voting. The HM makes the final call.
+- Interviewers submit feedback to TL+HM using the Google Form that TL attaches to your interview event.
```
Comment (tbaxter-18f) [inline _pages/detailed-process.md]: We do define TL and HM higher up in the doc, right?

Comment (ctro) [reply _pages/detailed-process.md]: :+1: Yep, we do.

**Pull request merged**